contents/newsletter/b2b-startup-marketing-strategy.md

```diff
@@ -0,0 +1,155 @@
+
+Above all, don’t get distracted by rivals doing something better than you. Stick to what you want to be great at.
+
+> **Further reading:** [How we spend our marketing budget (with actual $ figures)](https://posthog.com/founders/actual-marketing-budget)
+
+## 2. Compete on depth, not breadth 👨‍🎓
+
+Trying to go head-to-head with much larger competitors on content output is a losing game, so focus on quality. One great article >>> 25 generic ones. We write approx. ⅓ SEO articles, ⅓ tutorials, ⅓ anything goes.
+
+High quality articles perform better in the long run, especially SEO ones. Treat your SEO portfolio like a VC treats their investments – invest time updating your content, don’t publish and forget.
+
+## 3. Build a channel you truly own 💌
+
+Being really good at Twitter/X, LinkedIn, or writing blog posts for HackerNews is worthwhile, but the engagement you get is transient and often subject to the whims of opaque algorithms.
+
+It’s one of the key reasons we launched this newsletter. With the slight exception of Gmail’s “helpful” sorting features, no one other than you decides whether you see our latest issue. And we can be confident the time we invest building *Product for Engineers* won’t be blown to bits by factors outside our control.
+
+Other options include:
+
+- A really great Discord (better for communities than Slack)
+- Your own subreddit (not something we’ve tried)
+- A community on your website (we’re building it)
+- A podcast (good ones take way more effort than you think)
+
+Why do this? FWIW, a single *Product for Engineers* newsletter drives more clicks to our website than a $5,000 newsletter sponsorship we ran recently.
+
+## 4. Make your website genuinely different 🙃
+This means:
+
+1. Keeping it separate from marketing, which is the opposite of what most companies do.
+
+2. Avoiding popular design tropes among similar products – how many devtool websites have you seen that look just like Linear?
+
+![posthog website](../images/newsletter/startup-marketing/website.png)
+
+While our marketing and website teams work closely together, the website team has the final say on what appears on posthog.com. This stops the website becoming too marketing-y and falling foul of marketing ‘best practices’ that most developers (and people in general) despise.
+
+A good (and different) website is a great source of word-of-mouth.
+
+## 5. Perfect attribution is a white whale 🐳
+
+Here’s a simplified history of online marketing attribution:
+
+![ryan reynolds](../images/newsletter/startup-marketing/marketing.png)
+
+
+Attribution has always been hard, and it’s only getting harder because traditional methods are impeded by privacy regulations. This is a 7/10 problem. The journey from 7/10 to 10/10 will end you.
+
+**Here’s our simple advice:**
+
+1. Setup some basic tracking ([multi-touch attribution](https://www.linkedin.com/advice/1/what-pros-cons-using-first-touch-last-touch-multi-touch) is more of late-stage thing)
+
```

Comment:
```suggestion
```

I wouldn't space these lists out so much, here or elsewhere in the article. Why are we doing that? We don't do it elsewhere.